File: EduChemSuite.UI/UI/src/app/_models/ExamAssignmentModel.ts
import {BaseModel} from "./BaseModel";
import {ExamModel} from "./ExamModel";
import {UserModel} from "./UserModel";

export class ExamAssignmentModel extends BaseModel {
  examId?: string;
  exam?: ExamModel;
  userId?: string;
  user?: UserModel;
  assignedAt?: Date;
}

File: EduChemSuite.API/EduChemSuite.API/Models/ExamAssignmentModel.cs
namespace EduChemSuite.API.Models;

public class ExamAssignmentModel : BaseModel
{
    public required Guid ExamId { get; set; }
    public ExamModel? Exam { get; set; }

    public required Guid UserId { get; set; }
    public UserModel? User { get; set; }

    public DateTime? AssignedAt { get; set; }
}
